### ISSUE TITLE
[Bug]:  Dismissing a single notification is dismissing them all

### ISSUE THREAD

**Issue by wongcain** [OneSignal/OneSignal-Android-SDK#1697]
### What happened?

We recently have been seeing that dismissing a single message dismisses them all.  We are not currently setting a group id, and instead are allowing OneSignal to do it's auto-grouping that it does on Android 7+.
Looking at the release notes, it looks like 4.8.3 added a fix related to ungrouped notification dismissal.  The release date coincides with when we started seeing this issue.

In parallel to submitting this ticket, I am reverting back to 4.8.2 to see if it resolves the issue.  I will report back in response to this ticket once I verify.

### Steps to reproduce?

```Markdown
1. Wait until you have received enough notifications for them to be grouped.
2. Expand a group of notifications.
3. Swipe to dismiss one of the notifications in the group.
```

### What did you expect to happen?

I expect only the swiped notification  to be dismissed.  Instead all of the notifications in the group are dismissed.

Note that not all users on all devices are seeing this.

### OneSignal Android SDK version

4.8.3 

### Android version

13

### Specific Android models

```Markdown
Pixel 6

(This is the device and version we have seen it on so far.  I suspect it is not limited to this version and model however.)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by wongcain** [comment]
Here is a video a co-worker shared replicating the problem:  https://photos.app.goo.gl/mrks3xrMJiF4rQq38

**Comment by wongcain** [comment]
I have verified that this problem no longer occurs if I downgrade to version4.8.2

**Comment by nan-li** [comment]
Hi @wongcain,

Thanks for reporting this. It is indeed is a bug introduced when we fixed another bug and released in version 4.8.3.
We are now working on a fix for both.

In the meantime, downgrading to `4.8.2` is fine since `4.8.3` doesn't have any additional features.

**Comment by wongcain** [comment]
Are there any updates on this issue?

**Comment by nan-li** [comment]
Hi @wongcain we made the fixes in release 4.8.4, so you can update to that version or the latest version 4.8.5

**Comment by wongcain** [comment]
Thank you!
